docs/content/manual/v1.6/manual.yml

```diff
@@ -2421,6 +2421,11 @@ sections:
       - title: "Variable / Symbolic Binding Operator: `... as $identifier | ...`"
         body: |
 
+          The expression `exp as $x | ...` means: for each value of expression
```

Comment:
`s/expression/form/`

Reply:
The rest of this text looks fine, though perhaps a lot of the text following this new text needs to be altered or deleted.